contracts/standard/rng/BlockhashRNGFallbackRange.sol

```diff
@@ -30,12 +30,8 @@ contract BlockHashRNGFallback is BlockHashRNG {
      */
     function saveRN(uint _block) public {
         if (_block < block.number && randomNumber[_block] == 0) {// If the random number is not already set and can be.
-            if (blockhash(_block) != 0x0) { // Normal case.
-                randomNumber[_block] = uint(blockhash(_block));
-            }
-            else { // The contract was not called in time. Fallback to returning any one of the blockhash of last mined 256 blocks.
-                randomNumber[_block] = uint(blockhash((block.number-1) - (block.number-1-_block)%256));
-            }
+            // The below equation is valid for all cases (last 256 blockhash of blocks or even earlier than that)
```

Comment:
Need "."

Reply:
Done

Reply:
// Returns the blockhash of _block if accessible (one of the last 256 blocks). If the blockhash hasn't been saved in time, return a blockhash of a block in range as a fallback.

Reply:
Done